test/typescript-node/eslint.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
eslint test is still failing, can you correct the test by referring to my PR which added node 22 support to TS-node?